### PR TITLE
[IPCT1-245] - Add new failed community modal

### PR DESCRIPTION
**Pull request by Jonathansoufer**
This PR adds updated explanatory text in cases where community creation fails and adds a link to the create ticket docs.

WhatsApp Image 2021-07-07 at 10 36 01

### CODE REVIEW

**Review by obernardovieira — CHANGES_REQUESTED** [review]
Lets not overengineer and simply use two `const` variables with URL

File: src/views/createCommunity/index.tsx [hunk 1]
```diff
@@ -368,10 +369,10 @@ function CreateCommunityScreen() {
                 setIsEmailValid(false);
             }
 
-            const _isEnabledGPS = gpsLocation !== undefined;
-            if (!_isEnabledGPS) {
-                setIsEnabledGPS(false);
-            }
+            // const _isEnabledGPS = gpsLocation !== undefined;
```
Comment (obernardovieira) [inline src/views/createCommunity/index.tsx:372]: commented?

**Review by obernardovieira — CHANGES_REQUESTED** [review]
Images or video, please

File: src/views/createCommunity/index.tsx (hunk 1 above)
Comment (Jonathansoufer) [reply src/views/createCommunity/index.tsx:372]: Good eye. Forgot to undo this ... finished this too late ... Fixed.

**Comment by Jonathansoufer** [comment]
Using modal instead of modalize since the second option isn't working as supposed. 

WhatsApp Image 2021-07-08 at 09 15 49

**Review by obernardovieira — CHANGES_REQUESTED** [review]
We've discussed before, and this has been most requested by Afonso that, models should only be used for quick alerts.
According to the design, a webview must be within a new view, allowing the user to be back to the previous page. Lets follow the design.
image

**Pull request closed without merging**